application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -147,17 +175,24 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
             return;
         }
 
-        OptionMapping usedOption = (applicationOption != null) ? applicationOption : idOption;
 
-        handleSubcommand(event, voiceChannel, usedOption, maxUses, maxAge);
+        String ID;
+        if (applicationOption != null) {
+            ID = VC_APPLICATION_TO_ID.get(applicationOption.getAsString());
+        } else {
+            ID = idOption.getAsString();
+        }
```

Comment:
NIT: use ternary

Reply:
Disagree, find readability horrible due to the length of the if statement.